integration_test/third_party_apps_data/applications/mssql/metadata.yaml

```diff
@@ -38,11 +38,13 @@ expected_metrics:
   monitored_resource: gce_instance
   labels: {}
 # v2 metrics
-- type: workload.googleapis.com/sqlserver.user.connection.count # TODO: look into this metric type change
-  value_type: DOUBLE
+- type: workload.googleapis.com/sqlserver.user.connection.count
```

Comment:
For the records: we are ok with changing the expected type from DOUBLE to INT because the V2 mssql metrics was never released.